int16 pwm_ratio1
int16 pwm_ratio2
uint8  gear_ratio
uint8  all_wheel_drive
int16 SpeedSetPoint
time   time_received
